tools/development/k8s/memory/patch_cgroup_driver/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+# This experimental playbook is meant to be used as a workaround.
+
+- hosts: kubernetes_master:kubernetes_node
+  gather_facts: true
+  tasks: []
+
+- hosts: kubernetes_node
+  serial: 1
+  environment:
+    KUBECONFIG: /etc/kubernetes/admin.conf
+  tasks:
+    - name: Define cgroup drivers
+      set_fact:
+        old_cgroup_driver: >-
+          {{ old_cgroup_driver | default('cgroupfs') }}
+        new_cgroup_driver: >-
+          {{ new_cgroup_driver | default('systemd') }}
```

Comment:
```suggestion
      set_fact:
        old_cgroup_driver: cgroupfs
        new_cgroup_driver: systemd
```

Reply:
I moved this to `vars` and added the `rollback` procedure! :+1::innocent: